src/components/Navbar.jsx

```diff
@@ -11,12 +12,13 @@ function Navbar() {
   const sections = useMemo(
     () => [
       { id: 'landing', text: 'Landing' },
-      { id: 'speakers', text: 'Speakers' },
-      { id: 'sessions', text: 'Sessions' },
       { id: 'location', text: 'Location' },
+      { id: 'sessions', text: 'Sessions' },
       { id: 'sponsors', text: 'Sponsors' },
+      { id: 'jobboard', text: 'Job Board' },
```

Comment:
Possibly comment out the Job Board link until we implement that section

Reply:
Agreed with @ShugKnight24